services/src/main/scala/cromwell/pipeline/service/ProjectService.scala

```diff
@@ -3,13 +3,15 @@ package cromwell.pipeline.service
 import java.util.UUID
 
 import cromwell.pipeline.datastorage.dao.repository.ProjectRepository
-import cromwell.pipeline.datastorage.dto._
+import cromwell.pipeline.datastorage.dto.{ Project, ProjectAdditionRequest, ProjectId }
 import cromwell.pipeline.service.Exceptions.{ ProjectAccessDeniedException, ProjectNotFoundException }
 
 import scala.concurrent.{ ExecutionContext, Future }
 
 class ProjectService(projectRepository: ProjectRepository)(implicit executionContext: ExecutionContext) {
 
+  import cromwell.pipeline.datastorage.dto.{ ProjectUpdateRequest, UserId }
+
```

Comment:
I can't understand why you've added this import here instead of line 6. Can you explain it?

Reply:
Well, I don't know why, but when I do import something - this something importing in smallest scope. I tried catch this thing's but now I missed it. I missed it when changing imports. Fixed.